src/config/constants.js

```diff
@@ -32,6 +32,10 @@ const constants = {
           account
         })
     }
+  },
+  DISCORD: {
+    url: 'https://discord.gg/MRjKBXtaDt',
```

Comment:
Remove the `https:`

Reply:
the 'https:' needs to be written explicitly at someplace. It's most readable to be written here.
For the links to Voyager & Etherscan it's written in ChainType.js

Reply:
No need to define the browser protocol. It will automatically be opened in the bridge's protocol.

Reply:
done